### PR TITLE
Add paquo comment

### PR DESCRIPTION
**Pull request by alanocallaghan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,35 +1,52 @@
...
+    *without any direct dependency on QuPath and Java*.
+    It therefore doesn't require QuPath to be installed, and
+    can be used entirely from Python.
+
+QuBaLab doesn't share code with QuPath, but is uses many of the same conventions
+for accessing images and representing objects in a GeoJSON compatible way.
+By using the same custom fields for things like measurements and classifications,
+exchanging data is much easier.
+
+[paquo](https://paquo.readthedocs.io/en/latest/index.html) is an existing library
```
Comment (petebankhead) [inline README.md:31]: Proposed replacement section:

---

### How does QuBaLab compare to paquo?

[paquo](https://paquo.readthedocs.io) is an existing library linking Python and QuPath that provides a pythonic interface to QuPath.

_We think paquo is great, and don't want to replace it!_

Here are the 3 main differences as we see them:
1. **Target audience**
    * paquo is written mostly for Python programmers who need to work with QuPath data
    * QuBaLab is written mostly for QuPath users who want to dip into Python
2. **Convenience vs. Efficiency**
    * paquo is based on [JPype](https://jpype.readthedocs.io/) to provide full & efficient access to Java from Python
    * QuBaLab is based on [Py4J](https://www.py4j.org) to exchange data between Java & Python - preferring convenience over efficiency
3. **Pixel access**
    * paquo is for working with QuPath projects and objects - accessing pixels is [beyond its scope (at least for now)](https://forum.image.sc/t/paquo-read-write-qupath-projects-from-python/41892/26)
    * QuBaLab enables requesting pixels as numpy or dask arrays, and provides functions to convert between thresholded images & QuPath objects

So if you're a Python programmer who needs an intuitive and efficient way to work with QuPath data, use paquo.

But if you're a QuPath user who wants to switch to Python for some tasks, including image processing, you might want to give QuBaLab a try.

**Review by petebankhead — COMMENTED** [review]
Thanks, I've made some suggestions.

Would you object vehemently to standardising around semantic line breaks, across our readmes & docs...?

File: README.md [hunk 2]
```diff
@@ -65,6 +82,8 @@ pytest                                                  # to run unit tests
 ## OpenSlide support
 
 OpenSlide support relies on having OpenSlide binaries installed at a system
-level. Therefore not all aspects of OpenSlide will work consistently across
+level. Therefore, not all aspects of OpenSlide will work consistently across
 platforms, as the versions available from package managers or from OpenSlide
-directly may vary between operating systems.
+directly may vary between operating systems. We hope that this will be resolved
```
[the remark this answers is not in the document]
Comment (alanocallaghan) [reply README.md:87]: openslide-python 1.4.0 needs to be release, which will use the openslide-bin binaries. At the moment on Linux at least it loads the system 3.4.1 binaries

**Comment by alanocallaghan** [comment]
> Would you object vehemently to standardising around semantic line breaks, across our readmes & docs...?

I hate them, but if I'm in a minority then sure

**Pull request merged**